src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -463,6 +475,22 @@ export const TradeInput = ({ isCompact }: TradeInputProps) => {
 
   const walletSupportsBuyAssetChain = useWalletSupportsChain(buyAsset.chainId, wallet)
 
+  const nativeAssetBridgeWarning: string | [string, InterpolationOptions] | undefined =
+    useMemo(() => {
+      if (!buyAssetFeeAsset) return
+      // TODO(gomes): Bring me in for all bridges?
```

Comment:
Not part of the spec but might as well at some point? Might require some more work to ensure we detect bridges properly without false positives however.